torch/csrc/jit/passes/quantization.cpp

```diff
@@ -178,10 +178,10 @@ Node* insertObserver(
     Value* v,
     Graph* g,
     script::Module& module,
-    const QConfig& qconfig) {
+    const QConfig& qconfig,
+    const std::unordered_set<Value*>& weight_values) {
```

Comment:
Might be worth to refactor all that into a helper class to avoid passing these sets in every function. Later we'll also add a similar set for 'bias' values.

Reply:
OK will do refactor in a separate PR